cabal-install/cabal-install.cabal

```diff
@@ -375,6 +379,9 @@ test-suite integration-tests2
   hs-source-dirs: tests
   default-language: Haskell2010
 
+  other-modules:
+    IntegrationTests2.ProjectConfig.ParsecTests
+
```

Comment:
This change looks a bit dodgy. I think it's a general confusion we have among new contributors in that integration tests for `cabal-install` are supposed to go into `cabal-testsuite` (see it's readme for details) instead on `cabal-install/tests/IntegrateionTests2`. So, if it's not too big of a nuisance, I'd suggest to move there. But that's not critical, of course.

Reply:
Perhaps, I should have added that my understanding is that `cabal-install/tests/IntegrateionTests2` was deprecated and superseded by `cabal-testsuite`.

Reply:
Thanks for your hint! No problem, I did not know that IntegrationTests2 was deprecated. I will move the tests to the testsuite.

Reply:
I tried to move my tests into cabal-testsuite, but I am really not sure how. Those tests invoke the cabal binary directly and then compare the written cabal output to some expectation.
If you take a look into my ParsecTests (https://github.com/haskell/cabal/pull/8889/files#diff-beb0804e5838f281a9013a397de783c41055fdeab5eaec4d09e4dcfddde15a50), I do not call cabal, I just use plain Haskell functions to parse .project files and then compare the parsed value to some expectation, so cabal does not output anything here.

When thinking how to do it, I found the Cabal-tests module which contains ordinary unit tests. Maybe this module is more appropriate for my tests? It contains tests that are similar to mine already (https://github.com/haskell/cabal/blob/master/Cabal-tests/tests/ParserTests.hs). 
What do you think about this?